pagopa/io-app
# chore: [IOBP-981,IOBP-1002] Add PSP_ERROR outcome screen into payment flow

## Short description
This PR adds the PSP_ERROR screen inside the payment outcome flow.

## List of changes proposed in this pull request
- Mapped the `PSP_ERROR` outcome as value `25`;
- Mapped the `PSP_ERROR` to mixpanel analytics;
- Mapped the outcome `BE_NODE_KO` as `PAYMENT_99_ERROR` to mixpanel analytics;

## How to test
- Start a payment flow with the dev-server started;
- When you reach the final screen that opens inside the in-app browser, choose from the dropdown the `PSP_ERROR` value;
- Check that the expected screen is the same [as designed here](https://www.figma.com/design/OwkkVcUF05ZAfNkOp51Wwo/Flusso-di-Pagamento---Tracking-Strategy?node-id=1190-10188&t=Qdl1N46kVd5YdgW5-4)

## Preview

https://github.com/user-attachments/assets/e5ab3737-01f4-4347-be77-cbffcaeec073
